tsdat/io/readers/tar_reader.py

```diff
@@ -101,6 +102,9 @@ def read(self, input_key: str) -> Dict[str, xr.Dataset]:
                 continue
 
             for key in self.parameters.readers.keys():
+                # TODO: This could return `None`, but isn't specified as optional typing. Type
+                #  specification should be Optional[DataReader], where Optional is imported from
+                #  the typing library.
```

Comment:
yup, good catch

Reply:
Resolved in 854367387733e5508b772277e991bdf64b6ce373.